README.md

```diff
@@ -214,7 +214,7 @@ This section lists each configuration option, and whether it can be set by each
 
 | Purpose | CLI Flag | Env Var | Secret Field | Default |
 | --- | --- | --- | --- | --- |
-| Path to config secret |    |    | `cloud-config` | error |
+| Path to config secret | `cloud-config` |    |  | error |
```

Comment:
This change is a docs fix unrelated to the MetalLB issue.  I can break it out into a separate PR if needed.